src/inet/InetInterface.cpp

```diff
@@ -542,7 +543,7 @@ static struct if_nameindex * backport_if_nameindex(void)
         lastIntfName = addrIter->ifa_name;
     }
```

Comment:
Also use `MemoryFree` in lines 509 and 513 above

Reply:
Same these are not allocated by MemoryAlloc

Reply:
Aren't they? I think that `backport_if_freenameindex` is used to free resources allocated by `backport_if_nameindex`, so `MemoryFree` should be used there, right?

Reply:
I think you are right, the memory allocated by strdup is freed at 591 inside backport_if_nameindex